test/DynamoCoreWpfTests/PackageManagerUITests.cs

```diff
@@ -269,7 +269,8 @@ public void PackageManagerCrashTestOnDownloadingInvalidPackage()
             }
 
             var loader = GetPackageLoader();
-            Assert.AreEqual(loader.LocalPackages.Count(), 0);
+            var packageFound = loader.LocalPackages.Any(x => x.Name == "Autodesk Steel Connections 2020");
```

Comment:
not sure whats up with this - maybe something got reverted on master.

Reply:
I think this package is not meant to be installed as it is in a nested folder. @reddyashish would know.

Reply:
Yes, this package shouldn't be loaded as it is an invalid one. I think if you have have any other package with similar name in the packages folder, this fails?. do we want to change the name of this package?

Reply:
ahh, I see what happened, this is a fix I had sitting around for a while - instead of checking if no packages are loaded I changed this test so it checks that *this* package is not loaded. I wanted to make this change because I had other packages installed locally so the test failed.